azurelinuxagent/ga/policy/policy_engine.py

```diff
@@ -15,10 +15,22 @@
 # Requires Python 2.4+ and Openssl 1.0+
 #
 
+import json
+import os
 from azurelinuxagent.common import logger
 from azurelinuxagent.common.event import WALAEventOperation, add_event
 from azurelinuxagent.common import conf
 from azurelinuxagent.common.exception import AgentError
+from azurelinuxagent.common.protocol.extensions_goal_state_from_vm_settings import _CaseFoldedDict
+
+
+# Customer-defined policy is expected to be located at this path.
+# If there is no file at this path, default policy will be used.
+_CUSTOM_POLICY_PATH = conf.get_policy_file_path()
```

Comment:
do we still need this module-global variable?

Reply:
I kept it to easily accommodate changes to where we keep the policy path (for example, moving from osutil to conf). Can remove if you think it's not needed.

Reply:
I thought it was a left-over from a previous iteration, when we defining the value here.

Seems odd that we define this global for this conf parameter, while we don't for conf.get_extension_policy_enabled (and other parameters in the rest of the code)